frontend/states/partials/menu/subCat/sub_category.html

```diff
@@ -0,0 +1,55 @@
+<section>
```

Comment:
I like how dynamic this is however there is a lot of repeated lines. I would be cool if each li was in an ng-repeat i.e have an array in your controller with all the category names and repeat am `<li>` for each of them